omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -59,6 +59,12 @@
   notifies :restart, 'runit_service[opscode-erchef]' unless backend_secondary?
 end
 
+template "/opt/opscode/embedded/service/opscode-erchef/vm.args" do
+  source "oc_erchef.vm.args.erb"
```

Comment:
But the other `vm.args` I looked at was pretty different. I'll see if I can make some common template parts I will. If they are all different, I'll leave them as separate templates.